cmd/get_logs.go

```diff
@@ -72,14 +72,14 @@ func newGetLogsCmd() *cobra.Command {
 	}
 	command.Flags().StringVarP(&glc.location, "location", "l", "", "Azure location where the cluster is deployed (required)")
 	command.Flags().StringVarP(&glc.apiModelPath, "api-model", "m", "", "path to the generated apimodel.json file (required)")
-	command.Flags().StringVar(&glc.apiserverURI, "apiserver", "", "apiserver endpoint (required)")
-	command.Flags().StringVar(&glc.linuxSSHPrivateKeyPath, "linux-ssh-private-key", "", "path to a valid private ssh key to access the cluster's Linux nodes (required)")
-	command.Flags().StringVar(&glc.linuxScriptPath, "linux-script", "", "path to the log collection script to execute on the cluster's Linux nodes")
+	command.Flags().StringVar(&glc.sshHostURI, "sshHost", "", "FQDN, or IP address, of an SSH listener that can reach all nodes in the cluster (required)")
```

Comment:
One last nit! (I promise)

Let's make it `--ssh-host` instead of `-sshHost` to be more consistent with other > 1 word concatenation arg expressions.

Thanks for doing this!